test/gtest/uct/test_uct_perf.cc

```diff
@@ -50,6 +50,16 @@ class test_uct_perf : public uct_test, public test_perf {
              ++test_iter) {
             test_spec test = *test_iter;
 
+            if (has_transport("cuda_copy") || has_transport("gdr_copy")) {
+                test.send_mem_type = UCS_MEMORY_TYPE_HOST;
+                test.recv_mem_type = UCS_MEMORY_TYPE_CUDA;
+
+                test.max += 10.0;
```

Comment:
Maybe add a comment here about (1) what is the reason for this addition, and (2) why this value was chosen.